helpers/db.json

```
@@ -0,0 +1,66 @@
+[
+  {
+    "id": 1,
+    "article": "Body",
+    "price": 10,
```

I recommend to add decimal digits 👇
```suggestion
    "price": 10.00,
```